src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/Packaging/CSharpProjectSystemPackage.cs

```diff
@@ -13,6 +14,8 @@ namespace Microsoft.VisualStudio.Packaging
 {
     [Guid(PackageGuid)]
     [PackageRegistration(AllowsBackgroundLoading = true, RegisterUsing = RegistrationMethod.CodeBase, UseManagedResourcesOnly = true)]
+    [RemoteCodeGeneratorRegistration(ResXGenerators.ResXGuid, ResXGenerators.ResXGeneratorName, ResXGenerators.ResXGeneratorName, @"{" + ProjectTypeGuid + @"}", GeneratesDesignTimeSource = true)]
```

Comment:
Also, I'll add an attribute that allows us to register file extensions with a generator, so we get .tt support.